### DISCUSSION TITLE
@SuppressWarnings on code hints and NetBeans 16.

### DISCUSSION THREAD

**Discussion by jmccay-work** [apache/netbeans#5484] [Q&A] [answered]
Netbeans 12 had the option to in the code hints context menu (right click) to suppress warnings.  I can't seem to find that option in NetBeans 16.  Is it still available, and if so, how do I turn on the option.  I don't use it that much, but it is handy when I do use it.  Thanks you.

**Comment by mbien** [comment, marked as answer]
not all warnings can be suppressed. I just tried it with the `rawtypes` compiler warning and it worked: it added `@SuppressWarnings("unchecked")`.

Any warning in particular you want to ignore?

**Comment by jmccay-work** [comment]
Thanks.  I don't remember the specific warning that I was trying to suppress, but I remember it was annoying.  I have since seen ones that I could suppress through the context menu.

**Reply by Chris2011** [reply]
Suppress warning is not about the context manu as far as I can remember. it is about the hints and the hints are shown by alt + enter (default) so if you see a warning/error smth else where you can click on it or open it via alt + enter, often there should be the entry "Suppress warning". But as @mbien said, not all the time.
